.github/workflows/pack_publish.yml

```diff
@@ -192,3 +192,57 @@ jobs:
           artifact: build_windows_RelWithDebInfo_x64
           type: arc
           delimiter: "-"
+  vars:
+    runs-on: ubuntu-22.04
+    outputs:
+      app_version: ${{ steps.vars.outputs.app_version }}
+      app_date: ${{ steps.vars.outputs.app_date }}
+      app_commit: ${{ steps.vars.outputs.app_commit }}
+
+    steps:
+      - id: vars
+        run: |
+          version=$( cat CMakeLists.txt | grep 'columnar VERSION' | cut -d' ' -f5  )
+          date=$( date +%Y-%m-%d )
+          commit=${GITHUB_SHA:0:7}
+
+          echo "app_version=$version" >> $GITHUB_OUTPUT
+          echo "app_date=$date" >> $GITHUB_OUTPUT
```

Comment:
This looks worrisome since at midnight or if this workflow is delayed, the date may be wrong. It's better to take it from the commit. This https://github.com/manticoresoftware/columnar/blob/master/cmake/rev.cmake#L24-L25 may be helpful.